projects/project-player-library/src/lib/assets/i18n/en.json

```diff
@@ -65,14 +65,14 @@
     "TASK_DESCRIPTION":"Task description",
     "PROJECT_DOWNLOADING_SUCCESS":"You have downloaded the project successfully!",
     "CERTIFICATE_CRITERIA":"Certificate criteria",
-    "LEARNING_RESOURCES":"Learning resources",
+    "LEARNING_RESOURCES":"View resources",
```

Comment:
Give relevant key names, give it as "VIEW_RESOURCES"

Reply:
Done